fivox/densityImageSource.h

```diff
@@ -0,0 +1,62 @@
+
+/* Copyright (c) 2014-2016, EPFL/Blue Brain Project
+ *                          Daniel.Nachbaur@epfl.ch
+ *
+ * This file is part of Fivox <https://github.com/BlueBrain/Fivox>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef FIVOX_DENSITYIMAGESOURCE_H
+#define FIVOX_DENSITYIMAGESOURCE_H
+
+#include <fivox/imageSource.h>
+#include <fivox/types.h>
+
+namespace fivox
+{
+
+/** Image source iterating over all events to calculate density per voxel. */
```

Comment:
It provides two things:
- Consistency of a single color map across multiple datasets. This will make sense once Ahmet finishes the new color map GUI, where the control points have floating point coordinates in a free range.
- Consistency of the colormap across levels of detail. Which is not necessary when you have a single level of resolution, but that won't be always the case.

If you insist on keeping counts at least rename the class to a name which is not misleading.